MangoPay/PayOut.php

```diff
@@ -19,12 +31,24 @@ class PayOut extends Transaction
      */
     public $PaymentType;
 
+	/**
+	 * One of PayOutPaymentDetails implementations, depending on $PaymentType
+	 * @var \MangoPay\PayOutPaymentDetailsBankWire
+	 */
+	public $PayoutPaymentDetails;
+
     /**
      * One of PayOutPaymentDetails implementations, depending on $PaymentType
-     * @var object
+     * @var \MangoPay\PayOutPaymentDetailsBankWire
      */
     public $MeanOfPaymentDetails;
 
+	/**
+	 * Payout mode requested, default is 'STANDARD', Allowed values are
+	 * 'STANDARD', 'INSTANT_PAYMENT', 'INSTANT_PAYMENT_ONLY'
+	 * @var string
+	 */
+	public $PayoutModeRequested = 'STANDARD';
```

Comment:
```suggestion
	public $PayoutModeRequested;
```
if I am correct they never put default values in objects